spec/integration/inherited_commands_spec.rb

```diff
@@ -1,30 +1,31 @@
 # frozen_string_literal: true
 
+require "open3"
+
 RSpec.describe "Inherited commands" do
   context "when --help flag" do
     it "subclass do not" do
```

Comment:
this test name sounds incomplete, but I'm not sure what it could be

Reply:
Yeah, this is a bit perplexing. Let's change it to `it "includes inherited arguments in help output"`